test/requests/foods/create.test.js

```diff
@@ -61,5 +61,30 @@ var specHelper = require('../../specHelper');
             done();
           })
         })
+      it('returns "name must be unique" if the food is already in database', (done) => {
+        request(app)
+          .post('/api/v1/foods')
+            .send({
+              food: {
+                name: "Mint",
+                calories: 14
+              }
+            })
+            .then(() => {
+              request(app)
+                .post('/api/v1/foods')
+                  .send({
+                    food: {
+                      name: "Mint",
+                      calories: 14
+                    }
+                  })
+                  .then(response => {
+                    expect(response.body.error.errors[0].message).to.equal("name must be unique")
```

Comment:
Let's add an assertion for a particular status code (probably 422) and make sure the controller catches it appropriately. Also need to call out in the README that food names must be unique and state what response you'll get if you try do create a duplicate.